src/commands/DebuggerCommands.ts

```diff
@@ -70,7 +66,11 @@ async function getQuickPickItems(txProvider: TransactionProvider) {
   });
 }
 
-function generateDebugAdapterConfig(txHash: string, workingDirectory: string, providerUrl: string): DebugConfiguration {
+export function generateDebugAdapterConfig(
```

Comment:
Do we need to `export` this `function`? As I can see, it's only being used in this module.